docs/deployment/deployment.md

```diff
@@ -13,15 +13,15 @@ Begin by first forking the Information Assistant repository into your own reposi
 To fork the repo simply click the **Fork** button at the top of the Information Assistant Repo page and follow the steps to set up your new fork.
 ![Fork the Information Assistant Repo](/docs/images/fork_repo.png)
 
-Once you have forked the repo, you can then use the following button to open the Information Assistant Codespace. You will need to select your forked repo and the location for your Codespace to run in.
+Once you have forked the repo, you can then use the following button to open the Information Assistant GitHub Codespaces. You will need to select your forked repo and the location for your GitHub Codespaces to run in.
 
-[![Open in GitHub Codespaces](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=false&ref=main&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json)
+[![Open in GitHub Codespacess](https://img.shields.io/static/v1?style=for-the-badge&label=GitHub+Codespaces&message=Open&color=brightgreen&logo=github)](https://github.com/codespaces/new?hide_repo_select=false&ref=main&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fdevcontainer.json)
```

Comment:
Too many "S"s

Reply:
I still see two "S" in the link.. 
![image](https://github.com/microsoft/PubSec-Info-Assistant/assets/48474707/39b4e47d-c129-49fd-b5f6-999e108df890)